ntoskrnl/io/iomgr/driver.c

```diff
@@ -2181,4 +2194,237 @@ NtUnloadDriver(IN PUNICODE_STRING DriverServiceName)
     return IopUnloadDriver(DriverServiceName, FALSE);
 }
 
+/*
```

Comment:
Please use doxygen documentation format (for examples, see files in ntoskrnl/se/ )